visualization/app/codeCharta/assets/sample1.cc.json

```diff
@@ -1,7 +1,7 @@
 {
 	"projectName": "Sample Project with Edges",
 	"apiVersion": "1.2",
-	"fileChecksum": "invalid-md5-sample",
+	"fileChecksum": "valid-md5-sample1",
```

Comment:
Nit picking: Intentionally, "invalid-md5-sample" should express that the sample value is not a valid md5. So, "valid-md5-sample" is a bit confusing here. I agree that the former is not that obvious. Therefore, we could just omit the prefix "valid".